communication/slack-config/restrictions.yaml

```diff
@@ -13,4 +13,8 @@ restrictions:
   - path: "sig-docs/*.yaml"
     channels:
       - "^kubernetes-docs-[a-z]{2}$"
+  - path: "sig-release/*.yaml"
+    channels:
+      - "^sig-release$"
+      - "^release-managers$"
```

Comment:
Sort of opposite from before XD Do you want them to be that restrictive? It means in the future you'd have to open it up or add another explicit entry.

Reply:
Yeah, that was the goal. For the time being, I don't want arbitrary groups slipping in. We can reassess later if it makes senses to fragment the usual conversations further.

Reply:
Cool. Just wanted to double check.

/lgtm